src/libraries/System.Runtime/tests/System.Globalization.Tests/DateTimeFormatInfo/DateTimeFormatInfoShortDatePattern.cs

```diff
@@ -11,9 +11,9 @@ public class DateTimeFormatInfoShortDatePattern
         public static IEnumerable<object[]> ShortDatePattern_Get_TestData_HybridGlobalization()
         {
             // see the comments on the right to check the non-Hybrid result, if it differs
-            yield return new object[] { "ar-SA", "d\u200f/M\u200f/yyyy" }; // "d\u200f/M\u200f/yyyy g"
+            yield return new object[] { "ar-SA", PlatformDetection.IsHybridGlobalizationOnBrowser ? "d\u200f/M\u200f/yyyy" : "d MMM، yyyy g" }; // "d\u200f/M\u200f/yyyy g"
```

Comment:
utf-8 should be in encoded form in the code, they are more visible:
```suggestion
            yield return new object[] { "ar-SA", PlatformDetection.IsHybridGlobalizationOnBrowser ? "d\u200f/M\u200f/yyyy" : "d MMM\u060c yyyy g" }; // "d\u200f/M\u200f/yyyy g"
```